src/index.js

```diff
@@ -179,11 +186,10 @@ class ServerlessThundraPlugin {
                 const { layers } = provider
                 const func = slsFunctions[key]
                 const funcName = key
-                const runtime = func.runtime || provider.runtime
+                let runtime = func.runtime || provider.runtime
 
-                if (get(func, 'custom.thundra.disable', false)) {
-                    this.warnThundraDisabled(funcName)
-                    continue
+                if (runtime.startsWith('dotnetcore')) {
```

Comment:
This might be put after of isString check below. I don't know how isString check here is important in real life but if we have isString check already, and if runtime is not a string `runtime.startsWith`(...) call will fail. So I think it is better to have
```js
let runtime = func.runtime || provider.runtime

if (!isString(runtime)) {
    continue
}

if (runtime.startsWith('dotnetcore')) {
    runtime = 'dotnetcore'
}
```

Reply:
It has been fixed with commit [c419ef59](https://github.com/thundra-io/serverless-plugin-thundra/commit/c419ef5946699f87f79dec5f585d9c5798090dfe)